dstep/Configuration.d

```diff
@@ -77,6 +77,10 @@ struct Configuration
     /// do not translate following symbols
     @("skip", "Skip translation of <symbol>.")
     string[] skipSymbols;
+
+    /// enable printing of warnings
+    @("enable-warnings", "Enable printing of warnings [default].")
+    bool enableWarnings = true;
```

Comment:
Ok, it can be done (even `getopt` supports it) but there are two issues:

* how the option should be named
* there are more cases than `error`, `warning` and `ignore`: `error`, `warn_and_skip`, `warn_and_fix`, `ignore_and_skip` and `ignore_and_fix`.

My point is, that there are two orthogonal concepts: issuing a message and trying to fix the problem.